.github/pr-template-simple.md

```
@@ -0,0 +1,43 @@
+
+
+
+## 📌 작업 배경
+<!-- 왜 이 작업이 필요했는지 -->
+
+
+
+## 🔨 구현 내용
+<!-- 주요 변경사항을 간략히 나열 -->
+
+#### Added (추가)
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)